guides/hack/56-modules/07-advanced-features.md

```diff
@@ -0,0 +1,53 @@
+This section details some advanced semantics and behaviors of modules.
```

Comment:
Could you name this page 'closures and function pointers'? When users see 'advanced features' in the docs sidebar they won't know if the page is relevant to whatever question they have in mind.